lms/templates/courseware/courseware.html

```diff
@@ -14,6 +15,11 @@
 </%def>
 
 <%block name="bodyclass">view-in-course view-courseware courseware ${course.css_class or ''}</%block>
+
+<%block name="js_overrides">
```

Comment:
@muhammad-ammar - another interesting issue is that this living in courseware.html means that it doesn't work in courseware-chromeless.html which is the page the mobile app uses to render XBlocks. I think this might cause issues on mobile (FYI @nasthagiri @aleffert)

https://github.com/edx/edx-platform/blob/master/lms/templates/courseware/courseware-chromeless.html

I think we do need to move this into main.html so that it is available everywhere.